include/parser_interface.hpp

```diff
@@ -39,6 +39,21 @@ class ParserInterface
         return types::DbusVariantType();
     }
 
+    /**
+     * @brief API to write keyword's value on hardware.
+     *
+     * @param[in] i_paramsToWriteData - Data required to perform write.
+     *
+     * @return On success returns no.of bytes written on hardware, On failure
+     * returns -1.
+     */
+    virtual int writeKeywordValueOnHardware(
```

Comment:
As it is "readKeywordFromHardware" better to have "wroteKeywordOnHardware".
Value looks like an extra fitting,

Reply:
done